site/source/tutorials/9001_dojo1_dijits/index.md

```diff
@@ -0,0 +1,187 @@
+---
+layout: tutorial
+title: Using Dojo 1 Dijits
+overview: In this tutorial, you will learn how to use Dojo 1 Dijits in a Dojo 2 application.
+---
+
+{% section 'first' %}
+
+# Using Dojo 1 Dijits
+
+## Overview
+
+This tutorial will extend the the [Form widgets](../005_form_widgets/) tutorial.  In this tutorial, we will be taking some of the form widgets and replacing them with Dojo 1 Dijits.
```

Comment:
we will be taking some of the form widgets and replacing them with Dojo 1 Dijits. -> we will replace some of the Dojo 2 based form widgets and with Dojo 1 Dijits.